variant-switcher/src/ui.tsx

```diff
@@ -74,7 +99,27 @@ const App: React.FC = () => {
                     onChange={(e) => setToVariant(e.target.value)}
                     name={'The variant name to change into'}
                     value={toVariant}
+                    onKeyDown={(e) => {
+                        if (e.code === 'Enter') {
+                            submit();
+                        }
+                    }}
+                />
+            </div>
+            <div className={'checkbox-row'}>
+                <input
+                    type={'checkbox'}
+                    onChange={(e) => {
+                        setDeepSwap(e.target.value === 'true' ? 'false' : 'true');
+                    }}
+                    name={'swapChild'}
+                    value={deepSwap}
+                    checked={deepSwap === 'true'}
                 />
+                <label htmlFor={'swapChild'}>
+                    Deep swap
+                    <div className={'hint-text'}>Swap child layers after parent instances are swapped</div>
```

Comment:
Looking for a better way to say this. 

Let's say that a component called Parent is built using instances of another component Child. Think of it as: 

```tsx
const Parent = (
    <div>
        <Child />
    </div>
)
```

And both Parent and Child are built with a prop called Theme, which can be either Dark or Light.

Now suppose dark parent is built using light child, and light parent is built using dark child. Think about the snack bar that is dark in light theme and light in dark theme.

Like this:
```tsx
<Parent Theme={'Light'}> === <div><Child Theme={'Dark'} /></div>
<Parent Theme={'Dark'}> === <div><Child Theme={'Light'} /></div>
```
Under these rules, if we try to switch from Theme=Light to Theme=Dark
- When "Deep Swap" is checked, a "light parent dark child" would become "dark parent, dark child".
- When "Deep Swap" is unchecked, a "light parent dark child" would become "dark parent, light child".

So back to the wording here. I think "Deep swap" accurately captures what it does, but the explanation text here is necessary. I'd like to say something like "attempt to swap child layers / instances even after its parents have been swapped to the specified 'To Variant'", but I couldn't find the best way to re-word it.

Reply:
Just thinking about this a little bit, I would want to have options something like:

```
PropertyName: string;
FromVariant: string;
ToVariant: string;

DeepSwitch: boolean; // (false = only switch top level items, true = also switch any nested children)
TwoWay: boolean; // (false = only switch 'from'->'to', true = also switch 'to'->'from')
```

I'm thinking of the `TwoWay` input as changing between a 'set' mode and a 'toggle' mode. You could even call it `mode` with two options: 'set' and 'toggle' if you want.